assembl/static2/js/app/components/common/tree.jsx

```diff
@@ -280,13 +288,12 @@ class Tree extends React.Component {
               onResize={() => {
                 this.cache.clearAll();
                 this.listRef.recomputeRowHeights();
-                document.dispatchEvent(rowHeightRecomputed);
+                if (this.rootRef) this.rootRef.dispatchEvent(rowHeightRecomputed);
               }}
             >
               {({ width }) => {
                 return (
                   <List
-                    key={data.length}
```

Comment:
why did you removed the key on `List`? (it may leads to strange bugs)

Reply:
woops, did not meant to remove it. Because data.length is not a key? what if two List have the same length ? why not just put 0 ?

Reply:
it is indeed a strange value for a key. In case of 2 columns we could have 2 List with the same length so it may not be a good idea. I think that it was originally meant to ensure that if the size of the List changes, we re-render it.

Reply:
but the rowCount prop is already data.length

Reply:
So I just let the key removed right ?
The List is not an element of an array so we shouldn't need the key prop and data.length is passed as an other prop so the rerender behaviour does not change